driver/csiplugin/controllerserver.go

```diff
@@ -443,8 +441,8 @@ func (cs *ScaleControllerServer) createFilesetBasedVol(ctx context.Context, scVo
 		return filesetPath, nil
 	} else if scVol.VolumeType == cacheVolume {
 		createDataDir := false
-		klog.Infof("[%s] creating a fileset for a cache volume, fileset name: [%s] in filesystem [%s]", loggerId, scVol.VolName, scVol.VolBackendFs)
-		filesetPath, err := cs.createFilesetVol(ctx, scVol, scVol.VolName, fsDetails, opt, createDataDir, false, isCGVolume, bucketInfo)
+		klog.Infof("[%s] creating a fileset for a cache volume, fileset name: [%s] in filesystem [%s] and gatewayNodeName [%s]", loggerId, scVol.VolName, scVol.VolBackendFs, gatewayNodeName)
```

Comment:
Instead of saying `gatewayNodeName` in log here, better to say `gateway for export map` (to avoid confusion as the actual gateway used for fileset can be different than the one in export map)

Reply:
done